Add session storage support with function option (#1)
ryangjchandler/fern: README.md

</div>
```

> To use `sessionStorage`, or another custom storage that uses the same syntax, in place of `localStorage`, simply pass an optional third argument containing the storage object when creating the store.

Reviewer: ```suggestion
To use `sessionStorage`, or another custom storage that has an identical API, simply pass an optional third argument containing the storage object when creating the store.
```
